modules/ocf_www/manifests/lets_encrypt.pp

```diff
@@ -19,7 +19,10 @@
       command     => 'chronic /usr/local/bin/lets-encrypt-update -v web',
       user        => ocfletsencrypt,
       environment => ['MAILTO=root', 'PATH=/bin:/usr/bin:/usr/local/bin'],
-      special     => hourly,
+      # Run 5 minutes past the hour to allow for build-vhosts to be run so as
+      # to minimize the time between a vhost being configured and getting HTTPS
+      # enabled
+      minute      => 5,
```

Comment:
do we not need `hour => *` as well?

Reply:
No, it should be the default for all cron resources: https://github.com/ocf/puppet/blob/12cd03d03485b160f5b4f35efa351cc49f8bbd86/manifests/site.pp#L47